README.md

```diff
@@ -46,7 +46,7 @@ You can also grab the binary for your system from the [releases page](https://gi
 Alternatively, install with Go:
 
 ```bash
-go install github.com/aquasecurity/tfsec/cmd/tfsec@latest
+go get github.com/aquasecurity/tfsec/cmd/tfsec@latest
```

Comment:
Go get has been deprecated in favour of `go install ...`

Reply:
```
go install github.com/aquasecurity/tfsec/cmd/tfsec@latest                                                                                                                            
go: downloading github.com/aquasecurity/tfsec v0.58.15
go: downloading github.com/liamg/tml v0.4.0
go: downloading github.com/spf13/cobra v1.2.1
go: downloading github.com/hashicorp/go-version v1.3.0
go: downloading github.com/inconshreveable/go-update v0.0.0-20160112193335-8152e7eb6ccf
go: downloading github.com/liamg/clinch v1.5.6
go: downloading github.com/liamg/gifwrap v0.0.6
go: downloading github.com/owenrumney/go-sarif v1.0.12
go: downloading github.com/spf13/pflag v1.0.5
go: downloading golang.org/x/crypto v0.0.0-20210421170649-83a5a9bb288b
go: downloading golang.org/x/net v0.0.0-20210405180319-a5a99cb37ef4
go: downloading github.com/gdamore/tcell/v2 v2.2.0
go: downloading github.com/lucasb-eyer/go-colorful v1.0.3
go: downloading github.com/gdamore/encoding v1.0.0
```

What version of Go are you using?

Reply:
Can't check now but it's possible that outdated version is used on my side (it's in our CI pipeline, so I'm not sure what version is used there). So, if outdated Go version is the reason then feel free to close the PR. Thanks!

Reply:
Just checked:

```shell
$ go version
go version go1.14.7 linux/amd64
```